GitCommands/Git/Executable.cs

```diff
@@ -220,6 +221,27 @@ public StreamReader StandardError
                 }
             }
 
+            /// <inheritdoc />
```

Comment:
[nit] This is not required, the intellisense is smart enough to look up the docs when inheriting.